data/404/330/971/404330971.geojson

```diff
@@ -72,12 +72,13 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "esp-icgc:la_lc":8041,
```

Comment:
Missing a leading `0`, please add this back in a string (for all records).

Please also file followup issue for gleaning more Spanish "FIPS" codes from the QS props (similar to how you've done here).

Reply:
Fixed in 5c7c225